sdk/keyvault/keyvault-certificates/test/public/CRUD.spec.ts

```diff
@@ -120,11 +120,12 @@ describe("Certificates client - create, read, update and delete", () => {
   it("can update the tags of a certificate", async function (this: Context) {
     const certificateName = testClient.formatName(`${prefix}-${this!.test!.title}-${suffix}`);
 
-    await client.beginCreateCertificate(
+    const certificatePoller = await client.beginCreateCertificate(
```

Comment:
You would need to re-record this, and if it doesn't resolve the live tests in china cloud then you can probably just revert this and create an issue for the failing tests (assign it to me and I will look into it as soon as possible)

Reply:
The timeout problem no longer exists on China Cloud, thank you for your help.